end-to-end-test/shared/specUtils.js

```diff
@@ -191,6 +191,17 @@ function waitForNetworkQuiet(timeout) {
     }, timeout);
 }
 
+function closeDevModeBanner() {
```

Comment:
i think this is fine although a cleaner way might be to hide it using css alone.   we have a special stylesheet called e2etest.scss which gets applied only in e2e context.  You could hide it that way. If you go down this route, i would try to give that alert a specific class, or use a specific selector.  like ```body > div:n-th(0) .alert``` so as not to mess with other alerts

Reply:
btw the selector i put up there is just psuedo.

Reply:
I will open a separate PR for this since the changes need to be done at both frontend/backend repos.

Reply:
https://github.com/cBioPortal/cbioportal/issues/7561